src/components/helpers.ts

```diff
@@ -41,5 +41,8 @@ export function calculateWinner(squares: string[]) : string | null {
   if (squares[2] === squares[4] && squares[4] === squares[6]) {
     return squares[2];
   }
-  return null
+  if (squares.every((element: string) => element !== '-')) {
+    return null;
```

Comment:
Is this the tie case? If so we might want to just return 'tie' to make it extremely clear. 

As someone reading this code, it isn't super obvious the difference between returning `null` and `''`. Having it return `tie` will just make the logic easier to follow. What do you think?

Reply:
This is actually the case where if all the squares don't equal to '-', then return null.

Every square starts out as '-', so until all squares are clicked, null is returned.

Reply:
Makes sense-- we're checking this to basically see if it's a `tie` though, right? AKA, the entire board is filled out, but there is no winner